### PR TITLE
Fixes #15345: Updating the shared-files & remote-run APIs documentation

### PR DESCRIPTION
**Pull request by ArmandSy**
https://issues.rudder.io/issues/15345

### CODE REVIEW

File: src/reference/modules/reference/pages/relay_api.adoc [hunk 1]
```diff
@@ -1,102 +1,207 @@
...
 
 == Remote Run
 
-The remote run API is available at `https://relay/rudder/relay-api/remote-run`.
-It allows triggering a run on nodes (like with the `rudder remote run` command).
+The remote run API is available at
+`+https://relay/rudder/relay-api/remote-run+`. It allows triggering a
+run on nodes (like with the `+rudder remote run+` command).
 
-=== Description
```
Comment (amousset) [inline src/reference/modules/reference/pages/relay_api.adoc]: should the === here :)

**Comment by ArmandSy** [comment]
Commit modified

**Comment by ArmandSy** [comment]
Commit modified

File: src/reference/modules/reference/pages/relay_api.adoc [hunk 2]
```diff
@@ -1,102 +1,207 @@
...
 
-The relay that receive a *HEAD* call will:
+The metadata of the file is a text file containing key-values separated by a line feed, containing the following key-values pairs:
 
-* If the file exists, compare the provided hash with the hash of the stored file, and return the result (*true* or *false*).
-* If the file does not exist, return *false*.
+* `+header+` = _rudder-signature-v1_: signing format (for now, only one
+possible value)
+* `+algorithm+` = _sha256_ or _sha512_: hash algorithm used for signature
+* `+digest+` = **: signature of the file
```
Comment (amousset) [inline src/reference/modules/reference/pages/relay_api.adoc]: can you detail this? Which kind of signature, how is it done?

**Comment by ArmandSy** [comment]
Commit modified

**Pull request closed without merging**